index.js

```diff
@@ -3,4 +3,109 @@ function getGreeting (firstName, Surname) {
     return 'Hi, ' + '"' + firstName + ' ' + Surname + '".' + " What's up?";
 }
 const greeting = getGreeting('John', 'Smith');
-console.log(greeting); // Hi, “John Smith”. What's up?
\ No newline at end of file
+console.log(greeting); // Hi, “John Smith”. What's up?
+
+// Implement the isThisMyName function that returns true if provided with your name. Otherwise,
+// return false.
+function isThisMyName(name) {
+    return name === 'Marcin';
+    // if (name === 'Marcin') {
+    //     return true;
+    // } else {
+    //     return false;
+    // }
+}
+let isMyName = isThisMyName('Adam');
+console.log(isMyName);
+isMyName = isThisMyName('Marcin');
+console.log(isMyName);
+
+
+
+// 3. Implement the isThisBoolean function that returns true if provided with a boolean.
+
+function isThisBoolean(bool){
+    if (bool === true) {
+        return true;
+    } else if (bool === false) {
+        return true;
+    } else {
+        return false;
+    }
+}
+const boolean1 = isThisBoolean(true); //true
+console.log(boolean1);
+const boolean2 = isThisBoolean(false); // true
+console.log(boolean2);
+const boolean3 =isThisBoolean('true'); // false
+console.log(boolean3)
+
+// 5. Implement the getCircleArea function
+function getCircleArea(radius) {
+    return Math.PI * radius ** 2;
+}
+const radius = 5;
```

Comment:
Make sure the arguments in functions don't have the same names as variables outside of the function.